### ISSUE TITLE
chrono dep pulls in time version with CVE-2020-26235

### ISSUE THREAD

**Issue by LunNova** [frank2/exe-rs#7]
See chronotope/chrono/issues/602

Fix is probably to remove the "oldtime" feature from the `chrono` dep. The default features are `["clock", "std", "oldtime", "wasmbind"]`

```toml
chrono = { version = "0.4", features = ["clock", "std", "wasmbind"], default-features = false }
```

**Comment by frank2** [comment]
Holy crap a Rust CVE! Thanks for bringing this to my attention and offering the fix. Grateful that it appears to just be a DoS and I have limited use of the chrono crate.

**Comment by frank2** [comment]
Fixed as of this commit: https://github.com/frank2/exe-rs/commit/3a33fc15b13526cabad63d31a0f638f72cfe91fb This will be officially released in 0.5.5.

**Comment by frank2** [comment]
This is now available in 0.5.6, not 0.5.5! It's now live! Thanks again!
